xml/templates/civicrm_navigation.tpl

```diff
@@ -309,7 +309,7 @@ INSERT INTO civicrm_navigation
 VALUES
     ( @domainID, 'civicrm/admin/custom/group?reset=1',      '{ts escape="sql" skip="true"}Custom Fields{/ts}', 'Custom Fields',                             'administer CiviCRM', '',   @CustomizelastID, '1', NULL, 1 ),
     ( @domainID, 'civicrm/admin/uf/group?reset=1',          '{ts escape="sql" skip="true"}Profiles{/ts}', 'Profiles',                                       'administer CiviCRM', '',   @CustomizelastID, '1', NULL, 2 ),
-    ( @domainID, 'civicrm/admin/tag?reset=1',               '{ts escape="sql" skip="true"}Tags (Categories){/ts}', 'Tags (Categories)',                     'administer CiviCRM', '',   @CustomizelastID, '1', NULL, 3 ),
+    ( @domainID, 'civicrm/tag?reset=1',               '{ts escape="sql" skip="true"}Tags (Categories){/ts}', 'Tags (Categories)',                     'administer CiviCRM', '',   @CustomizelastID, '1', NULL, 3 ),
```

Comment:
in PR #9126 @seamuslee001 asked if permission should be "manage tags" here.

Answer:
I don't know...
This menu/navigation entry is in the "admin" menu that is, for all the rest, with "administer CiviCRM" permission. Setting it to "manage tags" would not change anything since it would not appear anyway if parent "admin" menu is not displayed (cf "administer CiviCRM" permission).
There are two logics and I don't know which is preferred by core maintainers:

    "manage tags" as for the other navigation entry pointing to this page (from Contact menu)
    "administer CiviCRM" as all other navigation entry from the admin menu